src/test/java/com/foundation/salesforce/runner/RunCukesTest.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * @(#) RunCukesTest.java Copyright (c) 2019 Jala Foundation.
+ * 2643 Av. Melchor Perez de Olguin, Colquiri Sud, Cochabamba, Bolivia.
+ * All rights reserved.
+ *
+ * This software is the confidential and proprietary information of
+ * Jala Foundation, ("Confidential Information").  You shall not
+ * disclose such Confidential Information and shall use it only in
+ * accordance with the terms of the license agreement you entered into
+ * with Jala Foundation.
+ */
+
+package com.foundation.salesforce.runner;
+
+import com.foundation.salesforce.core.report.GeneratorReport;
+import cucumber.api.CucumberOptions;
+import cucumber.api.testng.AbstractTestNGCucumberTests;
+import org.testng.annotations.AfterTest;
+
+
+/**
+ * RunCukesTest class
+ *
+ * @author Cristian Lujan
+ * @version 1.0
```

Comment:
Please put point at the to your version.

Reply:
Erhmmm, AFAIK I guess there's no need to place a dot at the the of @version